src/libstd/collections/hash/set.rs

```diff
@@ -544,10 +544,7 @@ impl<T, S> HashSet<T, S>
     ///
     /// If the set did not have a value present, `true` is returned.
     ///
-    /// If the set did have this key present, that value is returned, and the
-    /// entry is not updated. See the [module-level documentation] for more.
-    ///
-    /// [module-level documentation]: index.html#insert-and-complex-keys
+    /// If the set did have this key present, `false` is returned.
```

Comment:
It removes the guarantee, that the key is not updated, added in https://github.com/rust-lang/rust/pull/27894.
"If the set did have this key present, the key is not updated and `false` is returned." would be more correct.
Also, `BTreeSet` has identical docs, they should probably be updated too.